docs/en_US/CommunitySharings/AutoCompletion.md

```diff
@@ -18,7 +18,7 @@ For now, auto-completion will not be enabled by default if you install NNI throu
 cd ~
 wget https://raw.githubusercontent.com/microsoft/nni/{nni-version}/tools/bash-completion
 ```
-Here, {nni-version} should by replaced by the version of NNI, e.g., `master`, `v1.9`. You can also check the latest `bash-completion` script [here](https://github.com/microsoft/nni/blob/master/tools/bash-completion).
+Here, {nni-version} should by replaced by the version of NNI, e.g., `master`, `v1.9`. You can also check the latest `bash-completion` script [here](https://github.com/microsoft/nni/blob/v1.9/tools/bash-completion).
```

Comment:
use 1.9 doc in master branch?

Reply:
we can keep it for now, because master branch will be used for developing v2.0 which may have large changes

Reply:
well, how about set v1.9 as the default branch for users?